### PR TITLE
(chore) - Add pull request template for the project.

### PR DESCRIPTION
**Pull request by devinercolano**
Issue #20023 requested a template for pull requests

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+#Checklist
+- [] For PRs that fix an issue, reviewers should try reproduce the issue and verify that the pull request actually fixes it. Authors can help with this by including a unit test that fails without (and passes with) the change.
+
+- [] For PRs adding new features, reviewers should attempt to use the feature and comment on how it feels to use it. Example: if a PR adds a new command line flag, use the program with the flag and comment on whether the flag feels useful.
+
+- [] We expect appropriate unit test coverage. Reviewers should verify that new code is covered by unit tests.
+
+- []Code submitted must pass all unit tests and static analysis (“lint”) checks. We use Travis CI to test code on Linux, macOS and AppVeyor to test code on Microsoft Windows.
+
+- [] For failing CI builds, the issue may not be related to the PR itself. Such failures are usually related to flakey tests. These failures can be ignored (authors don’t need to fix unrelated issues), but please file a GH issue so the test gets fixed eventually.
```
Comment (ligi) [inline .github/PULL_REQUEST_TEMPLATE.md]: flakey -> flaky

**Comment by ligi** [comment]
Also a link to this: https://geth.ethereum.org/docs/developers/code-review-guidelines should be in there

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (holiman) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
- [] For failing CI builds, the issue may not be related to the PR itself. Such failures are usually related to flaky tests. These failures can be ignored (authors don’t need to fix unrelated issues), but please file a GH issue so the test gets fixed eventually.
```

**Comment by adamschmideg** [comment]
Superseded by #20243 because the original author didn't allow to make changes to her repository.

**Pull request closed without merging**